global.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Updated the Azure pipeline config - anything else?

Reply:
Yes, your change in `azure-pipelines.yml` is correct. But it would be enough to keep `5.0.200` in `global.json`. See [rollForward](https://docs.microsoft.com/de-de/dotnet/core/tools/global-json?tabs=netcore3x#rollforward) (latestPatch).

Reply:
Interesting, had some issues when loading the project. I'll try to revert the change here but I'd keep the latest version in the Azure pipeline, no reason to keep the old one I guess?

Reply:
>Interesting, had some issues when loading the project

That's on purpose if you do not have a compatible patch Version. This will prevent different behaviors with different Versions of .NET Core. This file pins the exact version.

>  I'll try to revert the change here

I have added some changes to your forked branch by pull request. I included that one as well.

> no reason to keep the old one I guess?

Yep, we can update the pipeline to a newer version.